### DISCUSSION TITLE
Welcome to money4band Discussions!

### DISCUSSION THREAD

**Discussion by MRColorR** [MRColorR/money4band#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OlivierVerhoek** [comment]
Hi, I am trying to run the stack on a RPi Zero 2. My first observations are:
The speed of the SBC is slow 30 down 30 up. (My RPi 4 8 gb wired runs 500 down and 65 up. Would this impact any earnings?

When I run the whole stack the CPU gets 62 isch celsius. It's trothling I gues at 80, so no issue yet. (Main btw)
Will try extra apps also soon.

I'll post some extra outcomes soon.

**Reply by MRColorR** [reply]
Wow, that's great 😄,
The low SBC should not bother the standard non extra apps too much. I'm a little concerned about the extra apps but maybe it will work fine too.
Also it should not enter throttling under 72-75°c but keep an eye on it an if possible increase its airflow. 

Keep us posted 😉

**Reply by OlivierVerhoek** [reply]
All selected containers by your software (non extra apps) are perfect for the RPi Zero 2. They runs smooth with very low CPU and memory use. Except for PacketStream. This container is to CPU intensive. What is see in the yaml file is image: packetstream/psclient:latest where the RPi Zero 2 might need a ARM version?
